storage/config.go

```diff
@@ -22,9 +22,18 @@ package storage
 type Config struct {
 	BBolt BBoltConfig `koanf:"bbolt"`
 	Redis RedisConfig `koanf:"redis"`
+	SQL   SQLConfig   `koanf:"sql"`
 }
 
 // DefaultConfig returns the default configuration for the module.
 func DefaultConfig() Config {
-	return Config{}
+	return Config{
+		SQL: SQLConfig{
+			ConnectionString: "file::memory:?cache=shared",
```

Comment:
What happens if I use a filename? Where is it stored? (not in data is my guess).
It is missing sane defaults, checks and documentation.